.github/workflows/widget-ci.yml

```
@@ -5,7 +5,7 @@ on:
 
   pull_request:
     paths:
-      - cumulocity-asset-properties-widget/asset-properties-widget/**
+      - cumulocity-asset-properties-widget/**
```

```suggestion
      - asset-properties-widget/**
```
use the correct folder path to look for changes. please refer documentation : https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths